src/main/java/com/r307/arbitrader/service/ticker/StreamingTickerStrategy.java

```diff
@@ -52,7 +52,7 @@ public void getTickers(Exchange stdExchange, List<CurrencyPair> currencyPairs, T
         if (!tickers.containsKey(exchange)) { // we're not receiving prices so we need to (re)connect
             ProductSubscription.ProductSubscriptionBuilder builder = ProductSubscription.create();
 
-            currencyPairs.forEach(builder::addTicker);
+            currencyPairs.forEach(pair -> { builder.addTicker(exchangeService.convertExchangePair(exchange, pair)); });
```

Comment:
This is a good find. Unfortunately I can't merge this PR until after we're ready to go to 5.0.6 and I won't be able to merge with the extra snapshot repository enabled. If you can reduce the PR to just this one change I could merge it today.

Reply:
Ok thanks. Just let you know that without 5.0.6 at least Binance doesn't work for me.
But I guess this line is separate from that issue.

Reply:
@scionaltera done!

Reply:
The line you fixed will still fix the bug in the code as it is now, and you're still free to use 5.0.6 in your own branch. We just haven't had a chance to test the upgrade to 5.0.6 _by itself_ much yet, and I keep merging things on top of #326 and #359 which is slowing testing on those big PRs. I'd like to get them out the door, bump to our next version and unblock everything else.